### PR TITLE
use Qt-install-action to install qt on windows

### PR DESCRIPTION
**Pull request by sithlord48**
Use the qt install action for the windows runners

### CODE REVIEW

File: src/gui/CMakeLists.txt [hunk 1]
```diff
@@ -259,3 +273,4 @@ if(INPUTLEAP_BUILD_TESTS)
         target_link_libraries(guiunittests Qt::Core5Compat)
     endif()
 endif()
+
```
Comment (p12tic) [inline src/gui/CMakeLists.txt:276]: Unnecessary change.

File: src/gui/CMakeLists.txt [hunk 2]
```diff
@@ -234,6 +234,20 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     install(TARGETS input-leap DESTINATION ${INPUTLEAP_BUNDLE_BINARY_DIR})
 elseif (${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "BSD")
     install (TARGETS input-leap DESTINATION bin)
+elseif(WIN32)
+    set(CMAKE_INSTALL_SYSTEM_RUNTIME_DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
+       set(CMAKE_INSTALL_UCRT_LIBRARIES TRUE)
+       include(InstallRequiredSystemLibraries)
+       add_custom_command(
+               TARGET input-leap POST_BUILD
```
Comment (p12tic) [inline src/gui/CMakeLists.txt:242]: Inconsistent indentation.

**Review by p12tic — CHANGES_REQUESTED** [review]
LGTM except a couple of nits.

**Comment by p12tic** [comment]
Please prefix commit descriptions with `github: `, `gui: `, `lib: ` depending on what's being changed.

File: src/gui/CMakeLists.txt (hunk 1 above)
Comment (sithlord48) [reply src/gui/CMakeLists.txt:280]: The file lacked a empty new line at the eof

Comment (p12tic) [reply src/gui/CMakeLists.txt:280]: Now it's two newlines :)
